packages/ckeditor5-font/src/ui/colorui.ts

```diff
@@ -63,6 +64,8 @@ export default class ColorUI extends Plugin {
 	 */
 	public colorTableView: ColorTableView | undefined;
 
+	private undoStepBatch: Batch;
```

Comment:
This prop is private (and it should be), so it should start with an underscore. Also, it missed API docs.